File: CommunicationModel/src/BrokerModels/CollectorPullEvent.cs
namespace CommunicationModel.BrokerModels
{
	public class CollectorPullEvent : ServiceEvent
	{

		public string sensorAddress { get; set; }
		public bool success { get; set; }
		public int returnedCount { get; set; }

		public CollectorPullEvent(
				string serviceId,
				string sensorAddress,
				bool success,
				int returnedCount,
				string additionalDesc = "")

				: base(serviceId, ServiceType.DataCollector, additionalDesc)
		{
			this.sensorAddress = sensorAddress;
			this.success = success;
			this.returnedCount = returnedCount;
		}

	}
}

File: DashboardClient/TBoardRabbitAdapter/src/main/java/broker/messages/incoming/CollectorPullEvent.java
package broker.messages.incoming;

public class CollectorPullEvent extends ServiceEvent {

	public String sensorAddress;
	public boolean success;
	public int returnedCount;

	public CollectorPullEvent(
			String sourceId,
			ServiceType type,
			String customMessage,
			String sensorAddress,
			boolean success,
			int returnedCount) {
		super(sourceId, type, customMessage);

		this.sensorAddress = sensorAddress;
		this.success = success;
		this.returnedCount = returnedCount;

	}

}
